### PR TITLE
Updates - Documentation / Python3 / TravisCI

### PR DESCRIPTION
**Pull request by markdessain**
Updating the docstrings and README, making sure its compatible with Python 2 and Python 3, updating the examples and finally adding in CI with Travis.

Builds for Python 2.7 and 3.5 can be found here: https://travis-ci.org/markdessain/mondo-python

Also I wasn't sure why the class was called client and was defined in the old style 

```
class client():
```

So I changed it to

```
class MondoClient(object):
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,6 +3,16 @@
 
 VERSION = '0.0.1'
 
+
+tests_requires = [
+    'nose>=1.3.4',
+    'responses>=0.5.1'
+]
+
+install_requires = tests_requires + [
```
Comment (icio) [inline setup.py]: Should `install_requires` include nose and responses?

**Comment by BenjaminEHowe** [comment]
"Also I wasn't sure why the class was called client and was defined in the old style" - my bad for not changing from the old style. I believe it was named client as if it's named MondoClient you end up with double-namespacing:

```
import mondo
ben = mondo.MondoClient(vars)
```

I mean, you could do `from mondo import MondoClient`, but if you wanted a mondo namespace (which I think you should, PEP 20 agrees and the mondo module isn't particularly heavyweight) presumably you would've just imported as-per my example above?

**Comment by markdessain** [comment]
Honestly it looked like it was committed by mistake, the confusion came about as the documentation said one thing but the code said something else. If you copied the README example it failed.

This commit https://github.com/simonvc/mondo-python/commit/5f9c2d0a753197d293bed577526e6c29f4cbfc9f didn't update the README.

I've no problem with it been changed back to Client(object). Just as long as it is consistent :+1:

**Comment by BenjaminEHowe** [comment]
You're right, I should've updated the README.

Currently snowed under with various uni deadlines, but at some point I'll submit a PR...

**Comment by markdessain** [comment]
I've been meaning to send some more additional PRs as well but been rather busy too.

Maybe having it a little tidier and a few extra features before the next hackathon would be a good goal.

**Comment by BenjaminEHowe** [comment]
Indeed. I'm hoping to find some time over the Easter vacation, but that might be ambitious given I'll be working FT for 2 weeks and revising for exams...

**Pull request merged**